modules/machineset-label-gpu-autoscaler.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * machine_management/applying-autoscaling.adoc
+// * machine_management/creating_machinesets/creating-machineset-aws.adoc
+// * machine_management/creating_machinesets/creating-machineset-azure.adoc
+// * machine_management/creating_machinesets/creating-machineset-azure-stack-hub.adoc
+// * machine_management/creating_machinesets/creating-machineset-bare-metal.adoc
+// * machine_management/creating_machinesets/creating-machineset-gcp.adoc
+// * machine_management/creating_machinesets/creating-machineset-ibm-cloud.adoc
+// * machine_management/creating_machinesets/creating-machineset-ibm-power-vs.adoc
+// * machine_management/creating_machinesets/creating-machineset-nutanix.adoc
+// * machine_management/creating_machinesets/creating-machineset-osp.adoc
+// * machine_management/creating_machinesets/creating-machineset-vsphere.adoc
+
+
+:_mod-docs-content-type: PROCEDURE
+[id="machineset-label-gpu-autoscaler_{context}"]
+= Labeling GPU machine sets for the cluster autoscaler
+
+You can use a machine set label to indicate which machines the cluster autoscaler can use to deploy GPU-enabled nodes.
+
+.Prerequisites
+* Your cluster uses a cluster autoscaler.
+
+.Procedure
+
+* Add a `cluster-api/accelerator` label to the machine set that you want to create machines for the cluster autoscaler to use to deploy GPU-enabled nodes:
++
+[source,yaml]
+----
+apiVersion: <api_version> <1>
+kind: MachineSet
+metadata:
+  name: machine-set-name
+spec:
+  template:
+    spec:
+      metadata:
+        labels:
+          cluster-api/accelerator: nvidia-t4 <2>
+
+----
+<1> The value of this parameter in the custom resource (CR) depends on whether the machine set uses Cluster API or Machine API.
```

Comment:
i need to study the cluster-api bits a little more, i have a feeling we will need 2 examples. looking at the metadata propagation rules, https://cluster-api.sigs.k8s.io/developer/architecture/controllers/metadata-propagation.html?highlight=propag#metadata-propagation , i'm not sure this works in cluster-api currently.

Reply:
Removed CAPI ref for the time being, as per discussion